it still lets the VM crash

```bash
[4](https://github.com/DPS25/bddbench/actions/runs/21800971377/job/64125172452?pr=112#step:5:115)
  multi-bucket write benchmark run (duration based) -- @1.3   E
--------------------------------------------------------------------------------
ERROR in step 'a SUT InfluxDB v2 endpoint is configured and reachable' (features/influx_multi_bucket_write_benchmark.feature:8):
Traceback (most recent call last):
  File "/run/github-runner/dsp25/bddbench/bddbench/bddbench/.venv/lib/python3.14t/site-packages/behave/model.py", line 1991, in run
    match.run(runner.context)
    ~~~~~~~~~^^^^^^^^^^^^^^^^
  File "/run/github-runner/dsp25/bddbench/bddbench/bddbench/.venv/lib/python3.14t/site-packages/behave/matchers.py", line 105, in run
    self.func(context, *args, **kwargs)
    ~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "features/steps/influx_global_endpoint_steps.py", line 12, in step_bucket_from_env
    raise RuntimeError("SUT InfluxDB endpoint is not reachable")
RuntimeError: SUT InfluxDB endpoint is not reachable

--------------------------------------------------------------------------------
  __CAPTURED_SCENARIO_OUTPUT__________________________
  CAPTURED LOG: before_scenario
  2026-02-19 09:32:55,254 DEBUG bddbench.environment: -- starting scenario: multi-bucket write benchmark run (duration based) -- @1.3
  ----
  CAPTURED LOG: after_scenario
  2026-02-19 09:33:05,271 DEBUG bddbench.environment: -- finished scenario: multi-bucket write benchmark run (duration based) -- @1.3  -> error
  __CAPTURED_SCENARIO_OUTPUT_END______________________
----
CAPTURED LOG: before_feature
2026-02-19 09:32:24,071 DEBUG bddbench.environment: === starting feature: InfluxDB v2 multi-bucket write benchmark (/api/v2/write) ===
----
CAPTURED LOG: after_feature
2026-02-19 09:33:05,272 DEBUG bddbench.environment: === finished feature: InfluxDB v2 multi-bucket write benchmark (/api/v2/write) -> error ===
---- CAPTURED_SCENARIO_OUTPUT_END ----

Error: Process completed with exit code 1.
```

--- a/features/influx_multi_bucket_query_benchmark.feature
+++ b/features/influx_multi_bucket_query_benchmark.feature
@@ -15,10 +15,10 @@ Feature: InfluxDB v2 multi-bucket query benchmark (/api/v2/query)
     Examples:
       | id    | measurement          | time_range | query_type | result_size | bucket_prefix     | bucket_count | concurrent_clients | output_format | compression |
       | smoke | bddbench_multi_write | 10s        | filter     | small       | bddbench_mb_smoke | 3            | 2                  | csv           | none        |
-      | load  | bddbench_multi_write | 1h         | aggregate  | large       | bddbench_mb_load  | 5            | 2                  | csv           | gzip        |
-      | stress| bddbench_multi_write | 1h         | aggregate  | large       | bddbench_mb_stress| 8            | 3                  | csv           | gzip        |
-      | spike | bddbench_multi_write | 5m         | filter     | small       | bddbench_mb_spike | 5            | 5                  | csv           | none        |
-      | soak  | bddbench_multi_write | 24h        | aggregate  | large       | bddbench_mb_soak  | 5            | 5                  | csv           | gzip        |
+      | load  | bddbench_multi_write | 1h         | aggregate  | large       | bddbench_mb_load  | 3            | 2                  | csv           | gzip        |
+      | stress| bddbench_multi_write | 1h         | aggregate  | large       | bddbench_mb_stress| 5            | 3                  | csv           | gzip        |
+      | spike | bddbench_multi_write | 5m         | filter     | small       | bddbench_mb_spike | 3            | 5                  | csv           | none        |
+      | soak  | bddbench_multi_write | 24h        | aggregate  | large       | bddbench_mb_soak  | 3            | 3                  | csv           | gzip        |
 
     @experimental
     Examples:

--- a/features/influx_multi_bucket_write_benchmark.feature
+++ b/features/influx_multi_bucket_write_benchmark.feature
@@ -16,11 +16,11 @@ Feature: InfluxDB v2 multi-bucket write benchmark (/api/v2/write)
     @normal
     Examples:
       | id    | measurement           | bucket_prefix      | bucket_count | batch_size | parallel_writers | compression | precision | point_complexity | tag_cardinality | time_ordering | duration_s |
-      | smoke | bddbench_multi_write  | bddbench_mb_smoke  | 3            | 100        | 1                | none        | ns        | low              | 10              | in_order      | 10         |
-      | load  | bddbench_multi_write  | bddbench_mb_load   | 5            | 250        | 2                | none        | ns        | medium           | 100             | in_order      | 20         |
-      | stress| bddbench_multi_write  | bddbench_mb_stress | 10           | 500        | 3                | none        | ns        | high             | 500             | in_order      | 45         |
-      | spike | bddbench_multi_write  | bddbench_mb_spike  | 10           | 250        | 8                | none        | ns        | medium           | 100             | in_order      | 10         |
-      | soak  | bddbench_multi_write  | bddbench_mb_soak   | 5            | 250        | 2                | none        | ns        | medium           | 100             | in_order      | 300        |
+      | smoke | bddbench_multi_write  | bddbench_mb_smoke  | 2            | 100        | 1                | none        | ns        | low              | 10              | in_order      | 10         |
+      | load  | bddbench_multi_write  | bddbench_mb_load   | 3            | 250        | 1                | none        | ns        | medium           | 100             | in_order      | 15         |
+      | stress| bddbench_multi_write  | bddbench_mb_stress | 5            | 400        | 2                | none        | ns        | high             | 250             | in_order      | 20         |
+      | spike | bddbench_multi_write  | bddbench_mb_spike  | 5            | 250        | 4                | none        | ns        | medium           | 100             | in_order      | 10         |
+      | soak  | bddbench_multi_write  | bddbench_mb_soak   | 3            | 250        | 1                | none        | ns        | medium           | 100             | in_order      | 60         |
 
     @experimental
     Examples:

--- a/features/influx_query_benchmark.feature
+++ b/features/influx_query_benchmark.feature
@@ -17,8 +17,8 @@ Feature: InfluxDB v2 query benchmark (/api/v2/query)
       | id    | measurement           | time_range | query_type | result_size | concurrent_clients | output_format | compression |
       | smoke | bddbench_single_query | 10s        | filter     | small       | 5                  | csv           | none        |
       | load  | bddbench_single_query | 1h         | aggregate  | large       | 10                 | csv           | gzip        |
-      | stress| bddbench_single_query | 1h         | aggregate  | large       | 20                 | csv           | gzip        |
-      | spike | bddbench_single_query | 5m         | filter     | small       | 50                 | csv           | gzip        |
+      | stress| bddbench_single_query | 1h         | aggregate  | large       | 15                 | csv           | gzip        |
+      | spike | bddbench_single_query | 5m         | filter     | small       | 25                 | csv           | gzip        |
       | soak  | bddbench_single_query | 24h        | aggregate  | large       | 10                 | csv           | gzip        |
 
     @experimental
@@ -26,7 +26,7 @@ Feature: InfluxDB v2 query benchmark (/api/v2/query)
       | id        | measurement           | time_range | query_type | result_size | concurrent_clients | output_format | compression |
       | smoke     | bddbench_single_query | 10s        | filter     | small       | 5                  | csv           | gzip        |
       | load      | bddbench_single_query | 1h         | aggregate  | large       | 10                 | csv           | gzip        |
-      | stress    | bddbench_single_query | 1h         | aggregate  | large       | 100                | csv           | gzip        |
-      | spike     | bddbench_single_query | 5m         | filter     | small       | 200                | csv           | gzip        |
+      | stress    | bddbench_single_query | 1h         | aggregate  | large       | 50                 | csv           | gzip        |
+      | spike     | bddbench_single_query | 5m         | filter     | small       | 100                | csv           | gzip        |
       | soak      | bddbench_single_query | 7d         | aggregate  | large       | 20                 | csv           | gzip        |
-      | breakpoint| bddbench_single_query | 10m        | filter     | small       | 300                | csv           | gzip        |
+      | breakpoint| bddbench_single_query | 10m        | filter     | small       | 150                | csv           | gzip        |

